nss-day-cohort-39/tabloid-full-stack-bobs
# Bobs add tags to post

### To Test:
- Go to post list, click to view a post's details
- Scroll to the bottom and select "Manage Tags"
- A modal should appear with a list of the tags currently associated with the post, and a list of tags not currently associated with the post
- Clicking a non-associated tag should remove it from the non-associated tags list, add it to the list of associated tags, and refresh each list
- Clicking an associated tag should remove it from the associated tags list, add it to the list of non-associated tags, and refresh each list
- Clicking the Save Tags button should close the modal and return the user to the post details
